cmd/goal/clerk.go

```diff
@@ -1262,25 +1230,73 @@ var simulateCmd = &cobra.Command{
 	Short: "Simulate a transaction or transaction group with algod's simulate REST endpoint",
 	Long:  `Simulate a transaction or transaction group with algod's simulate REST endpoint under various configurations.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		data, err := readFile(txFilename)
-		if err != nil {
-			reportErrorf(fileReadError, txFilename, err)
+		txProvided := cmd.Flags().Changed("txfile")
+		requestProvided := cmd.Flags().Changed("request")
+		if txProvided == requestProvided {
+			reportErrorf("exactly one of --txfile or --request must be provided")
+		}
+
+		requestOutProvided := cmd.Flags().Changed("request-out")
+		outProvided := cmd.Flags().Changed("outfile")
```

Comment:
Blame it on me: this used to be `--outfile` for result output, now I think it should be `"result-out"`?

Reply:
Thank you, I missed this